dev/com.ibm.ws.security.oauth/src/com/ibm/ws/security/oauth20/web/RegistrationEndpointServices.java

```diff
@@ -664,14 +609,9 @@ private static String computeRegistrationUri(HttpServletRequest request, String
     private OidcOAuth20Client validateClientIdExists(String clientId, OidcOAuth20ClientProvider clientProvider) throws OidcServerException {
         OidcOAuth20Client client = clientProvider.get(clientId);
         if (client == null) {
-            String browserErrorMsg = Tr.formatMessage(tc, locales, "OAUTH_CLIENT_REGISTRATION_CLIENTID_NOT_FOUND", new Object[] { clientId });
-            String serverErrorMsg = TraceNLS.getFormattedMessage(RegistrationEndpointServices.class,
-                    MESSAGE_BUNDLE,
-                    "OAUTH_CLIENT_REGISTRATION_CLIENTID_NOT_FOUND",
-                    new Object[] { clientId },
-                    "CWWKS1424E: The client id {0} was not found.");
-            Tr.error(tc, serverErrorMsg);
-            throw new OidcServerException(browserErrorMsg, OIDCConstants.ERROR_INVALID_CLIENT, HttpServletResponse.SC_NOT_FOUND);
+            BrowserAndServerLogMessage errorMsg = new BrowserAndServerLogMessage(tc, locales, "OAUTH_CLIENT_REGISTRATION_PUBLIC_CLIENT_NOT_FOUND", new Object[] { clientId });
```

Comment:
The message key in the update (`OAUTH_CLIENT_REGISTRATION_PUBLIC_CLIENT_NOT_FOUND`) doesn't match the original key (`OAUTH_CLIENT_REGISTRATION_CLIENTID_NOT_FOUND`).